src/containers/Tag/Tag.tsx

```diff
@@ -163,13 +163,9 @@ export const Tag: React.SFC<TagProps> = (props) => {
                 );
               })}
               <div className={styles.Buttons}>
-                <Button variant="contained" color="primary" onClick={submitForm}>
-                  Save
-                </Button>
+                <Button variant="contained" color="primary" onClick={submitForm} text="Save" />
```

Comment:
I think instead of passing text we can send the children

Reply:
Done with the changes @kurund have a look